subcontracted_service/models/warehouse.py

```diff
@@ -43,11 +43,10 @@ def _get_vals_for_proc_rule_subcontracting(self):
     @api.multi
     def _set_subcontracting_service_proc_rule(self):
         for rec in self:
-            if rec.subcontracting_service_proc_rule_id:
-                continue
-            vals = rec._get_vals_for_proc_rule_subcontracting()
-            rule = self.env['stock.rule'].create(vals)
-            rec.subcontracting_service_proc_rule_id = rule.id
+            if not rec.subcontracting_service_proc_rule_id:
```

Comment:
I don't understand this change. Please, revert it.

Reply:
It does the same without a continue.

Reply:
Yeah, I know. But it's unnecessary to add this code diff.

Reply:
4 lines against code diff from README is minimalistic